index.html

```diff
@@ -0,0 +1,154 @@
+            <div class="main-container">
+                <span class="main-program">Main program</span>
+                <hr class="hr-line">
+                <div class="program-img">
+                    <div class="lecture">
+                        <img src="images/program_icon_01.png" alt="">
+                        <p class="sub-heading">lecture</p>
+                        <span class="text">
+                            listen to speakers from differnt
+                             countries about the message of sharing and opening 
+                        </span>
+                    </div>
+                    <div class="lecture">
+                        <img src="images/program_icon_02.png" alt="">
+                        <p class="sub-heading">CC Exhibition</p>
+                        <span class="text">
+                            Apprecite various creations applying cc license of 
+                            artiist, oranised from art centre nabi korea 
+                        </span>
+                    </div>
+                    <div class="lecture">
+                        <img src="images/program_icon_03.png" alt="">
+                        <p class="sub-heading">Forum</p>
+                        <span class="text">
+                            make out time to share ideas and opinion with 
+                            experts for each topic in order to understand it
+                        </span>
+                    </div>
+                    <div class="lecture">
+                        <img src="images/program_icon_04.png" alt="">
+                        <p class="sub-heading">Workshop</p>
+                        <span class="text">
+                            try creating your own work using open 
+                            source license rather than just watching at it
+                        </span>
+                    </div>
+                    <div class="lecture">
+                        <img src="images/program_icon_05.png" alt="">
+                        <p class="sub-heading">CC Ignite</p>
+                        <span class="text">
+                            Get opportunities to network us CC affiliate around the 
+                            world and also after summiting 
+                        </span>
+                    </div>
+                </div>
+                <div class="btn-con"><button class="study-btn">join my study</button></div>
+                <li class="program-detail"><a href="#">See the whole program</a></li>
+            </div>
+        </section>
+        <section id="section2">
+            <div class="main-container">
+                <div class="presenters">
+                    <span class="main-program2">Main program</span>
+                    <hr class="hr-line">
+                    <div id="speakers-grid">
+
+                    </div>
+                </div>
+            </div>
+        </section>
```

Comment:
- [ ] Kindly follow the given design to align your speaker section, as it is currently not aligned like the given design.

Your design
![image](https://user-images.githubusercontent.com/81932478/196740663-2135e4df-292e-4e87-a355-305649b49c95.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/196740763-7d4a8431-d38f-454d-afc2-21372f796236.png)